eZ/Publish/Core/Persistence/Cache/LocationHandler.php

```diff
@@ -205,7 +205,14 @@ public function swap($locationId1, $locationId2)
 
         $return = $locationHandler->swap($locationId1, $locationId2);
 
-        $this->cache->invalidateTags(['location-data-' . $locationId1, 'location-data-' . $locationId2]);
+        $this->cache->invalidateTags(
+            [
+                'location-data-' . $locationId1,
+                'location-' . $locationId1,
+                'location-data-' . $locationId2,
+                'location-' . $locationId2,
```

Comment:
I think if you expire location, then there is no need to expire location-data which is a subset of location.

Reply:
Thanks, I suspected that, but was a bit unsure. Improved in 6d776e3